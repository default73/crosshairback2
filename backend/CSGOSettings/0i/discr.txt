ЧжиВэй "0i" Лю - киберспортсмен в дисциплине Counter-Strike: Global Offensive из Китая. Начал карьеру игрока в 206 году в составе коллектива FIVE, который и дал старт развития ЧжиВея как игрока. После него сменил несколько коллективов: New4, MFC, Invictus, с которым расстался в 2020 году и перешел в команду After, с которой выступал около 2 лет, а затем, в мае 2022 вернулся в уже знакомый ему Invictus.

Скачать 0i cfg csgo можно лишь у нас. Загрузка ои кфг ксго бесплатная.