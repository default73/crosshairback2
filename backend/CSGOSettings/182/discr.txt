Игнасио "182" Риус Вердуна - киберспортсмен в дисциплине Counter-Strike: Global Offensive из Аргентины. Начинал карьеру в составе малоизвестных южноамериканских коллективов. В 2021 начал играть в более именитых коллективах, начал с Leviatan в феврале. Так же выступал в Coscu Army, New Pampas, Hawks. В сентябре 2022 присоединился к FUSION, в составе которых выполняет роль рифлера.

Скачать 182 cfg csgo можно лишь у нас. Загрузка 182 кфг ксго бесплатная.