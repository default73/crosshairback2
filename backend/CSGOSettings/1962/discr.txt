Габриэль "1962" Синополи - профессиональный игрок в Counter-Strike: Global Offensive из Аргентины. Родился 19 ноября 1995 года. Начал карьеру киберспортсмена в 2016 году, первой командой для Габриэля стала Magnum Monsters. Так же имеет опыт выступления во многих коллективах из Южной Америки, в числе которых: Furious Gaming, Wild Ineters, Isurus Gaming, Coscu Army. С мая 2021 года состоит в команде Leviatán Esports, где выступает в качестве игрока основы.

Скачать 1962 cfg csgo можно лишь с нашего сайта. Загрузка 1962 кфг ксго бесплатная.