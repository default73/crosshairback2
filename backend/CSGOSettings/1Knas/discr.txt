Герман "1Knas" Хаттевиг - киберспортсмен в дисциплине Counter-Strike: Global Offensive из Швеции. Начинал карьеру игрока с выступлений в множестве малоизвестных локальных миксов. В 2021 году выступал в довольно известном среди семи-про игроков миксе Boys without Pyjamas, после которого его карьера пошла ввысь и он получил приглашение в состав коллектива 7AM, в котором выполняет роль рифлера. 

Скачать 1Knas cfg csgo можно лишь у нас. Загрузка 1кнас кфг ксго бесплатная.