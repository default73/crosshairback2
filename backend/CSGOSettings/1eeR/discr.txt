Александр "1eeR" Нагорный - профессиональный игрок в Counter-Strike: Global Offensive из Беларуси. Родился 31 декабря 2001 года в городе Лида. Начал карьеру киберспортсмена с миксов h4teU и shipachi. 21 февраля 2021 года присоединился к команде Zorka Gaming в качестве ин-гейм лидера. 19 августа 2021 года начал играть в коллективе Nemiga, а 30 августа было официально объявлено о том, что Александр присоединился к организации Nemiga Gaming на постоянной основе. В январе 2022 покинул их. В апреле побывал в коллективе PLATOON Alpha. 5 августа 2022 вернулся в расположение Nemiga Gaming.

Скачать 1eeR cfg csgo можно лишь с нашего сайта. Загрузка лир кфг ксго бесплатная.