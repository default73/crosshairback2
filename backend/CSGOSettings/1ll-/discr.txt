Илья "1ll-" Воронов профессиональный российский Counter-Strike: Global Offensive игрок. Родился 23 июня 2002 года. В конце 2017 года, серьезно начал увлекаться CS:GO. В 2020 Илья попадает в микс ANIMEplusW, с кем выходят на HLTV.

Скачать 1ll- cfg csgo можно лишь с нашего сайта. Загрузка илл кфг ксго бесплатная.