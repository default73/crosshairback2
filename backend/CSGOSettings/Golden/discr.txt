Майкил "Golden" Селим один из шведских игроков, который имеет иракское происхождение. С 2016 года он вытсупает на профессиональной Counter-Strike: Global Offensive сцене. Первой командой для него была visomvet. В августе 2016 попадает в академаю фнатиков. Через год ему удалось пробиться в основной состав Fnatic. Летом 2018 оказывается в рядах Cloud9, в конце года его переводят в инактив. Вернуться он смог лишь в марте 2019, но через три месяца вновь расстается с клаудами. С 17 сентября 2019 выступает на лане за Fnatic, а уже в январе был вновь переведен в основу фнатиков. 8 марта 2022 перешёл в команду Savage. 8 февраля 2023 перешёл в команду VELOX Argentina.

Скачать Golden cfg csgo можно лишь у нас. Скачивание Golden cfg cs go бесплатное, при скачивании голден кфг ксго не надо регистрироваться.